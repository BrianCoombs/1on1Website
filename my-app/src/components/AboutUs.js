import React, { Component } from 'react'

class AboutUs extends Component {

  render() {
    return (
      <div class="aboutUsBg" id="aboutus">
        <br/>
        <h1 >About Us</h1>
        <section id="aboutus">
        <div class="aboutUs">
          <br/>
          <p align="center">1on1 Tutoring is In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.</p>
          <div class="box">
            <img src="/img/checklist.jpeg" />
            <h4>Prepared</h4>
            <p>
              In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
            </p>
          </div>
          <div class="box">
            <img src="AboutUs/test.png" />
            <h4>Consistent</h4>
            <p>
              In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
            </p>
          </div>
          <div class="box" >
            <img src="/img/bulb.png" />
            <h4>Efficient</h4>
            <p>
              In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
            </p>
          </div>
        </div>
        </section>
      </div>
    );
  }
}

export default AboutUs;
