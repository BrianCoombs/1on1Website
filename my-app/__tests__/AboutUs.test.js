import React from 'react'
import { configure, shallow } from 'enzyme'
import AboutUs from '../src/components/AboutUs'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

configure({adapter: new Adapter()});

it('renders correctly', () => {
  const component = shallow(
    <AboutUs />
  );
  expect(toJson(component)).toMatchSnapshot();
});

/**
test('AboutUs component should render as expected', () => {
  const component = shallow(<AboutUs />)
  console.log(component)

});*/
