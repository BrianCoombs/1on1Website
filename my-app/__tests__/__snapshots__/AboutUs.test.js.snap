// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    id="aboutus"
  >
    About Us
  </h1>
  <section
    id="aboutus"
  >
    <div
      class="tutors"
    >
      <br />
      <p
        align="center"
      >
        1on1 Tutoring is In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
      </p>
      <div
        class="box"
      >
        <img
          src="/img/checklist.jpeg"
        />
        <h4>
          Prepared
        </h4>
        <p>
          In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </p>
      </div>
      <div
        class="box"
      >
        <img
          src="AboutUs/test.png"
        />
        <h4>
          Consistent
        </h4>
        <p>
          In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </p>
      </div>
      <div
        class="box"
      >
        <img
          src="/img/bulb.png"
        />
        <h4>
          Efficient
        </h4>
        <p>
          In mattis elit vitae odio posuere, nec maximus massa varius. Suspendisse varius volutpat mattis. Vestibulum id magna est.
        </p>
      </div>
    </div>
  </section>
</div>
`;
